### PR TITLE
PR: Add astunparse package

### PR DESCRIPTION
**Pull request by goanpeca**

### CODE REVIEW

File: recipes/astunparse/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - six >=1.6.1, <2.0
+    - wheel >= 0.23.0, <1.0
+
+test:
+  imports:
+    - astunparse
+
+about:
+  home: https://github.com/simonpercivall/astunparse
+  license: PSF
```
Comment (goanpeca) [inline recipes/astunparse/meta.yaml]: @jakirkham should this be BSD + PSF?

https://github.com/simonpercivall/astunparse/blob/master/LICENSE

Comment (jakirkham) [inline recipes/astunparse/meta.yaml]: Both I guess. Would include versions too (e.g. `BSD 3-Clause, PSF 2`).

File: recipes/astunparse/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - six >=1.6.1,<2.0
+    - wheel >=0.23.0,<1.0
```
Comment (synapticarbors) [inline recipes/astunparse/meta.yaml]: I think `wheel` isn't a real requirement of the package, but is instead part of the dev packaging requirements, so I think you can remove it.

**Comment by goanpeca** [comment]
@synapticarbors done :-)

**Pull request merged**